include/seqan/graph_msa/graph_align_tcoffee_library.h

```diff
@@ -833,6 +957,23 @@ _setDistanceValue(String<TFragment, TSpec>&,
 
 //////////////////////////////////////////////////////////////////////////////
 
+template<typename TScoreValues, typename TScoreValue, typename TSize>
+inline void
+_recordScores(TScoreValues & scores,
+              TScoreValue const & myScore,
+              TSize const & from,
+              TSize const & to)
+{
+    resize(scores, to);
+    typedef typename Iterator<TScoreValues, Standard>::Type TScoreIter;
+    TScoreIter itScore = begin(scores, Standard());
+    TScoreIter itScoreEnd = end(scores, Standard());
+    itScore += from;
+    for (; itScore != itScoreEnd; ++itScore) *itScore = myScore;
```

Comment:
Please put the for-loop body on the next line padded with 4 spaces at the beginning.